### PR TITLE
Snova jorge/multimodal base kit 

### PR DESCRIPTION
**Pull request by snova-jorgep**

### CODE REVIEW

**Comment by snova-amitk** [comment]
@snova-codym Can you help with a PR review for this multimodal kit from Jorge? Thanks!

File: multimodal_knowledge_retriever/notebooks/multimodal_rag.ipynb [hunk 1]
```diff
@@ -0,0 +1,786 @@
...
+    "from langchain_core.prompts import load_prompt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_prompt = load_prompt(os.path.join(kit_dir, \"prompts\", \"llama70b-text_summary.yaml\"))\n",
```
Comment (snova-codym) [inline multimodal_knowledge_retriever/notebooks/multimodal_rag.ipynb:398]: What are these yaml files?

Comment (snova-codym) [reply multimodal_knowledge_retriever/notebooks/multimodal_rag.ipynb:398]: Found below.  No worries.

File: multimodal_knowledge_retriever/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+pydantic_core==2.18.1
+python-dotenv==1.0.1
+langchain==0.1.17
+sentence_transformers==2.2.2
+instructorembedding==1.0.1
+streamlit-extras
+sseclient-py==1.8.0
+unstructured_inference==0.7.29
+unstructured[pdf]==0.13.6
+chromadb==0.4.24
```
Comment (snova-codym) [inline multimodal_knowledge_retriever/requirements.txt:12]: With python 3.9.12 I got:

ERROR: Could not find a version that satisfies the requirement onnxruntime>=1.17.0 (from unstructured-inference) (from versions: 1.7.0, 1.8.0, 1.8.1, 1.9.0, 1.10.0, 1.11.0, 1.11.1, 1.12.0, 1.12.1, 1.13.1, 1.14.1, 1.15.0, 1.15.1, 1.16.0, 1.16.1, 1.16.2, 1.16.3)
ERROR: No matching distribution found for onnxruntime>=1.17.0

No issues with Python dist 3.11.6.

We may need a note in the README.

**Review by snova-codym — APPROVED** [review]

**Pull request merged**